test/ModelTraitTest.php

```diff
@@ -49,23 +51,26 @@ public function testSetData($testData, $expectedProperties)
                 if (!empty($value["objectData"][0])) {
                     foreach ($value["objectData"] as $i => $objData) {
                         foreach ($objData as $subProp => $subValue) {
-                            $this->assertAttributeEquals($subValue, $subProp, $actualData[$i]);
+                            $assertGetter = "get" . ucfirst($subProp);
+                            $this->assertSame($subValue, $actualData[$i]->{$assertGetter}($subProp));
```

Comment:
The getter doesn't need an argument

Same on the change you made below

Reply:
Fixed